Makefile

```diff
@@ -33,11 +45,15 @@ cleanrun: clean run
 # Install requirements
 install:
 	python -m pip install --upgrade pip
-	pip install -r requirements.txt -r requirements-dev.txt
+	pip install -r requirements.txt
+
+install-dev:
```

Comment:
Install dev is missing the regular install. We could write it like this to extend install with Installation of dev files. 

```
install_dev: install
  Pip install req-dev....
````

Reply:
Good idea.
Do you prefer underscore (install_dev instead of install-dev) ?
To be consistent, I'll change run-container too.

Reply:
My fault.  Let's stick with "install-dev".